scripts/gis-material-update/README.md

```diff
@@ -0,0 +1,127 @@
+# Intro
+
+Download and process Haitaton spatial material (laastariaineisto).
+
```

Comment:
There could be a quick-start section or something here, which lists the actions the user has to take to make this work. In the current form, this can be limited to fetching the data.

Reply:
Thanks, created.